src/app/static/src/tests/components/surveyAndProgram/surveyAndProgram.test.ts

```diff
@@ -115,6 +138,11 @@ describe("Survey and programme component", () => {
         expect(choro.props().featureLevels).toBe("TEST LEVEL LABELS");
         expect(choro.props().indicators).toStrictEqual(["TEST INDICATORS"]);
         expect(choro.props().selections).toStrictEqual({selectedFilterOptions: "TEST SELECTIONS"});
+        expect(choro.props().colourScales).toEqual({prevalence: {
```

Comment:
In this case the warning was that the getter "selectedSAPColourScales" did not exist, which led me to notice that this prop was not being tested.